docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -88,7 +85,47 @@ gradient boosting methodologies.
 //end::regression-algorithms[]
 
 
-[discrete]
+[[dfa-regression-lossfunction]]
+===== Loss functions for {regression} analyses
+
+Loss function measures how well a given {ml} model fits the specific 
+data set. The loss function boils down all the different under- and 
+overestimations of the model to a single number. The bigger the difference 
+between the prediction and the ground truth, the higher the value of the loss 
+function. Loss functions are used automatically in the background during 
+<<hyperparameters,hyperparameter optimization>> and when training the decision 
+trees to compare the various iterations of the model and decide which iteration 
+performs better. 
+
+In the {stack}, there are three different types of loss function:
+
+* https://en.wikipedia.org/wiki/Mean_squared_error[mean squared error (`mse`)]: 
+It is the default choice when no additional information about the data set is 
+available.
+* mean squared logarithmic error (`msle`; a variation of `mse`): It is for 
+cases where the target values are all positive with a long tail distribution 
+(for example, prices or population).
+* https://en.wikipedia.org/wiki/Huber_loss#Pseudo-Huber_loss_function[Pseudo-Huber loss (`huber`)]:
+Use it when you want to prevent the model trying to fit the outliers instead of 
+fit regular data.
+
+The various types of loss function calculate the prediction error differently. 
```

Comment:
> ... calculate the prediction error differently...

This is the first time we've mentioned the term "prediction error". If this is the appropriate term for the number generated by loss functions, I think it should be introduced earlier. For example in the sentence "The loss function boils down ... to a single number, known as the prediction error".